sdk/dataprotection/Azure.ResourceManager.DataProtectionBackup/src/Generated/Models/SecureScoreLevel.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.DataProtectionBackup.Models
+{
+    /// <summary> Secure Score of Backup Vault. </summary>
+    public readonly partial struct SecureScoreLevel : IEquatable<SecureScoreLevel>
```

Comment:
rename SecureScoreLevel since it is taken in [RecoveryServices](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/recoveryservices/Azure.ResourceManager.RecoveryServices/src/Generated/Models/SecureScoreLevel.cs)

Reply:
Renamed